### ISSUE TITLE
[FEATURE] View shared mechs and pilots via the website

### ISSUE THREAD

**Issue by keyneston** [massif-press/compcon#2282]
**Is your feature request related to a problem? Please describe.**

The current method of sharing a mech or character on discord is to generate the statblock and then paste it into discord. Statblocks are nice, but are information dense and not the most pleasant way to view information.

**Describe the solution you'd like**

It would be nice if there was a way to generate a link to a pilot and/or mech. Then other people can open the link to view a read-only version of the pilot and/or mech. This way you have a nicer viewing interface than a stat block.

**Comment by keyneston** [comment]
> This can technically already be done by sharing a Share Code and importing the pilot as a remote resource, though, correct?

Technically yes you can. But that is a lot of steps to jump through to view someone's profile. Then afterwards you still need to delete it. Having a view option is much simpler.

**Comment by jarena3** [comment]
I like this idea and will incorporate it into v3

**Comment by msprijatelj** [comment]
Yeah honestly, this sounds like a good feature to have. Thinking about how D&D Beyond does character sheet sharing, it's just straightforward to grab a shareable link (and that link may even be a useful substitute for share codes).

**Comment by jarena3** [comment]
A shareable link is a nice idea -- it could load c/c and populate a readonly pilot sheet that you could then choose to save locally. It'd require a little router work but probably not a lot. I'll see if I can get a working version of this out in the v3 dev build for crit and feedback

**Comment by jarena3** [comment]
deployed in v3
